application-services/custom/camera-management/appcamera/commands.go

```diff
@@ -115,21 +116,31 @@ func (app *CameraManagementApp) sendPutCommand(deviceName string, commandName st
 		})
 }
 
-func (app *CameraManagementApp) getDevices() ([]dtos.Device, error) {
-	response, err := app.service.DeviceClient().DevicesByServiceName(context.Background(), app.config.AppCustom.DeviceServiceName, 0, -1)
-	if err != nil {
-		return nil, errors.Wrapf(err, "failed to get devices for the device service %s", app.config.AppCustom.DeviceServiceName)
-	}
-	if len(response.Devices) <= 0 {
-		return nil, errors.Errorf("no devices registered yet for the device service %s", app.config.AppCustom.DeviceServiceName)
+func (app *CameraManagementApp) getAllDevices() ([]dtos.Device, error) {
+	response1, err1 := app.service.DeviceClient().DevicesByServiceName(context.Background(), app.config.AppCustom.OnvifDeviceServiceName, 0, -1)
+	response2, err2 := app.service.DeviceClient().DevicesByServiceName(context.Background(), app.config.AppCustom.USBDeviceServiceName, 0, -1)
+
```

Comment:
Perhaps more meaningful names like onvifDevices and usbDevices instead of response1 and response2? may read better in later lines when overwriting or appending devices to the devices list.